pkg/types/baremetal/validation/platform.go

```diff
@@ -105,7 +105,7 @@ func ValidatePlatform(p *baremetal.Platform, n *types.Networking, fldPath *field
 
 	// Make sure the provisioning interface is set.  Very little we can do to validate this  as it's not on this machine.
 	if p.ProvisioningNetworkInterface == "" {
-		allErrs = append(allErrs, field.Invalid(fldPath.Child("provisioningNetworkInterface"), p.ProvisioningNetworkInterface, "No baremetal provisioning network interface is configured."))
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("provisioningNetworkInterface"), p.ProvisioningNetworkInterface, "no provisioning network interface is configured."))
```

Comment:
I wonder if we should make the error a bit more helpful, e.g say something like "no ProvisioningNetworkInterface is configured, this should be set to the interface name connected to the provisioning network on the deployed cluster nodes" ?

Reply:
Updated, good suggestion thanks